backend/services/implementations/authService.ts

```diff
@@ -16,12 +18,19 @@ class AuthService implements IAuthService {
 
   emailService: IEmailService | null;
 
+  firebaseAuth: firebaseAuth.Auth;
+
   constructor(
     userService: IUserService,
     emailService: IEmailService | null = null,
   ) {
     this.userService = userService;
     this.emailService = emailService;
+    const firebaseApp = firebase.initializeApp({
+      apiKey: process.env.FIREBASE_WEB_API_KEY,
+      projectId: process.env.FIREBASE_PROJECT_ID,
+    });
+    this.firebaseAuth = firebaseAuth.getAuth(firebaseApp);
```

Comment:
Are any of these Firebase things still being used in `AuthService`?

Reply:
If not, could we remove them from here + uninstall them (this should update `backend/package.json` and `backend/yarn.lock` as well)?